### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by reyazoct**

### CODE REVIEW

**Comment by ShubhamAgarwal1616** [comment]
Everything seems to be fine but try to avoid use of digits in commit messages

File: spec/matrix_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'matrix'
+
+describe "MatrixExtraction" do
+  matrix = MatrixExtraction.new
+  it "test to extract 0th row from '1 2\\n10 20'" do
```
Comment (abhi4890) [inline spec/matrix_spec.rb:5]: Avoid including test case data in test description

File: lib/matrix.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class MatrixExtraction
+  def row(matrix,rowValue)
```
Comment (abhi4890) [inline lib/matrix.rb:2]: Please use proper formatting

**Comment by swetavtiwari** [comment]
Looks good

**Pull request merged**